var/spack/repos/builtin/packages/wps/package.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+import glob
+import tempfile
+
+
+class Wps(Package):
+    """The Weather Research and Forecasting Pre-Processing System (WPS)
+    """
+
+    homepage = "https://www.mmm.ucar.edu/weather-research-and-forecasting-model"
+    url      = "https://github.com/wrf-model/WPS/archive/v4.2.tar.gz"
+
+    version('4.2', sha256='3e175d033355d3e7638be75bc7c0bc0de6da299ebd175a9bbc1b7a121acd0168')
+
+    # Serial variant recommended in WRF/WPS docs
+    variant('build_type', default='serial',
+            values=('serial', 'serial_NO_GRIB2', 'dmpar', 'dmpar_NO_GRIB2'))
+
+    # These patches deal with netcdf & netcdf-fortran being two diff things
+    patch('patches/4.2/arch.Config.pl.patch', when='@4.2')
+    patch('patches/4.2/arch.configure.defaults.patch', when='@4.2')
+    patch('patches/4.2/configure.patch', when='@4.2')
+    patch('patches/4.2/preamble.patch', when='@4.2')
+
+    # According to:
+    # http://www2.mmm.ucar.edu/wrf/users/docs/user_guide_v4/v4.0/users_guide_chap2.html#_Required_Compilers_and_1
+    # Section: "Required/Optional Libraries to Download"
+    depends_on('wrf')
+    depends_on('netcdf-c')
+    depends_on('netcdf-fortran')
+    # build script use csh
+    depends_on('tcsh', type=('build'))
+
+    # this fixes that for csh install scripts
+    depends_on('time', type=('build'))
+    depends_on('m4', type='build')
+    depends_on('libtool', type='build')
+    depends_on('jasper')
+    phases = ['configure', 'build', 'install']
+
+    def setup_build_environment(self, spack_env):
```

Comment:
Can you replace `spack_env` with `env` to be consistent with other packages?

Reply:
@adamjstewart 
Please mark me down as  official maintainer. 
also, "env" feedback resolved in latest push.

Reply:
To be a maintainer, simply add:
```python
maintainers = ['MichaelLaufer']
```
to the `package.py`. I usually put it just below the homepage/URL info. This can be a list of as many github handles as you want.